bin/cylc-jobscript

```diff
@@ -51,6 +51,16 @@ elif sys.argv[1] in ['-e', '--edit']:
 " "$1"
 }
 
+pager () {
+    if [[ -n $PAGER ]]; then
+        echo "$PAGER"
+    elif command -v less; then
+        echo less
+    else
+        echo cat
+    fi
+}
+
```

Comment:
Ditto comments above.

(Should really be in a common shell library) (but ... see final review comment...)